cogs/weightcog.py

```diff
@@ -557,6 +560,45 @@ async def export(self, ctx: commands.Context, user: discord.Member = None) -> No
                 file=discord.File(csv_file, filename=f"{user.display_name}_weight_data.csv"),
             )
 
+    @commands.command()
+    async def del_all_log(self, ctx: commands.Context, user: discord.Member = None) -> None:
```

Comment:
Don't  use abbreviations, also name itself is not descriptive about what it does. Alternative names:

delete_all_user_data
remove_weightcog_data

Reply:
I used an abbreviation one since the command:-
```
delete_all_user_data
remove_weightcog_data
```
will be too big.
If that's not a problem will replace with delete_all_user_data or i can pass the name parameter in ```    @commands.command()``` like
```
    @commands.command(name='something')
```
Suggestions on command name?

Reply:
That's perfectly fine to pass something in that decorator. But the function itself should have a clear name.

I still think that the command itself should be clear `del_all_log` is not very descriptive. If you have a better alternative let me know.

Reply:
I've changed the name to delete_all_user_data